public/components/content-list-drawer/content-list-drawer.js

```diff
@@ -29,6 +29,19 @@ export function wfContentListDrawer($rootScope, config, $timeout, $window, conte
         return config.composerRestorerUrl + '/' + composerId + '/versions';
     }
 
+    function isAtom(contentType) {
+        const allAtomTypes = [
```

Comment:
Possibly, but it's currently not there atoms are being added at a hell of a rate. perhaps something we should look at adding? Ideally, we;d have an 'atom' flag first then only use these to check for the parser to use.